### ISSUE TITLE
Change old PDF link to new Spreadsheet and update for 508 compliance

### ISSUE THREAD

**Issue by Jared-ONRR** [ONRR/onrr.gov-site#341]
Good Morning,

I have a file (PDF) that needs updated on ONRR.GOV to a new 508 compliant file (XLSX).  The direct link for the file is as follows: https://www.onrr.gov/ReportPay/PDFDocs/Gas_Plant_Listing.pdf

The link is found on https://www.onrr.gov/ReportPay/references.htm in the Manuals and Guidance section as •Gas Plant List for Onshore and Offshore (as of 10/15/2019).

That PDF needs replaced with the updated XLSX attached and noted as effective as of 10/15/2020.  

This is my first time requesting a website update, so please let me know if there are any additional information required or if there are any 508 issues with the file.

Thank you very much,

Jared Good 

[Gas_Plant_Listing.xlsx](https://github.com/ONRR/onrr.gov-site/files/5489995/Gas_Plant_Listing.xlsx)

**Comment by lpgoldstein** [comment]
@Jared-ONRR Thanks! This excel sheet isn't showing any accessibility errors so I will go ahead and replace the pdf with this file. I'll send you the updated page to review in a few minutes.

**Comment by lpgoldstein** [comment]
@Jared-ONRR this is ready for your review: http://2017onrr/ReportPay/references.htm

**Comment by Jared-ONRR** [comment]
I'm sorry, I incorrectly dropped an old revision of the file when I was working out the 508 compliance (only protected the Workbook and not the Sheet too).  Please replace with this file.  Other than that, the update looks good on the link you sent.

Thank you and sorry for the confusion. 

[Gas_Plant_Listing.xlsx](https://github.com/ONRR/onrr.gov-site/files/5490162/Gas_Plant_Listing.xlsx)

**Comment by lpgoldstein** [comment]
@Jared-ONRR the onrr.gov production server has been down since yesterday and we are unable to make any updates to the live site. I'll make the changes as soon as we are live again.

**Comment by Jared-ONRR** [comment]
Sounds good.  I'll keep an eye out for your update.

Jared


Jared Good
Minerals Revenue Specialist
Office of Natural Resources Revenue
Reference & Reporting Management
303-231-3319
________________________________

**Comment by lpgoldstein** [comment]
@Jared-ONRR It's live again - please review again http://2017onrr/ReportPay/references.htm

**Comment by Jared-ONRR** [comment]
Looks perfect!


Jared Good
Minerals Revenue Specialist
Office of Natural Resources Revenue
Reference & Reporting Management
303-231-3319
________________________________
